dbt_subprojects/hourly_spellbook/models/_sector/bridges/flows/bridges_flows.sql

```diff
@@ -0,0 +1,34 @@
+{{ config(
+    schema = 'bridges',
```

Comment:
all the schema here should be `bridges_evms.flows` etc., we will probably want to have a truely multichain bridges_crosschain.flows` eventually

Reply:
but what would that mean for ethereum>solana bridge transfers? they would be bound to be incomplete in both `bridges_evms.flows` and `bridges_solana.flows`, so i think it makes sense for `bridges_evms.deposits` and `bridges_evms.withdrawals` but probably not for the final `bridges.flows` model. wdyt?

Reply:
yeah that makes sense to me, but that would mean that we should set this up as  `bridges_crosschain.flows` and not use any varbinary columns to start with IMO

Reply:
i just moved it all into an `evms` subfolder and renamed `bridges.deposits/withdrawals/flows` to `bridges_evms.deposits/withdrawals/flows` and created a view for each on top, it might make sense to materialise those crosschain tables once bridge events from other vms are added

Reply:
alright, done